vaadin-spring/src/main/java/com/vaadin/flow/spring/VaadinServletContextInitializer.java

```diff
@@ -558,7 +558,8 @@ public VaadinServletContextInitializer(ApplicationContext context) {
             neverScan = Collections.emptyList();
         } else {
             neverScan = Arrays.stream(neverScanProperty.split(","))
-                    .map(String::trim).collect(Collectors.toList());
+                    .map(onlyFolder -> onlyFolder.replace('.', '/').trim())
```

Comment:
`onlyFolder` - what does that refer to?

Reply:
To use the folder instead of package (to be symmetrical to how `onlyPackages` is used with `vaadin.whitelisted-packages`)

Reply:
Though I know realize that "only" refers to the inclusion criterion of whitelisted packages.

Reply:
Changed to `pkg `